### DISCUSSION TITLE
Request: copy a kit row sound & all settings to new kit row

### DISCUSSION THREAD

**Discussion by thejesiah** [SynthstromAudible/DelugeFirmware#796] [Ideas]
Apologies if I missed this request, but surprised I didn't find it. Seems like it's one of the most common request on the FB group since the Deluge was released! 

Title basically says it - it would be a massive workflow enhancement / time saver to be able to copy/paste a kit row instrument sound to an empty kit row (within a single Kit or a different Kit). AKA duplicate kit row sound. All settings, waveforms points, etc. Allows for creating a bunch of variations quickly and easily, while maintaining settings for manipulation. 

It's been mentioned in the FB group that you can copy/paste via the XML. I hope it's just as easy for the firmware to do that! 

Thank you in advance to anyone who can bring this to life.

**Comment by therealstaubi** [comment]
I support this idea an would suggest to allow linking the settings of the original and the copied instrument (both in kit and song view). This would allow to tweak the sounds and transition smoothly to different rhythms/melodies at the same time.

**Comment by sichtbeton** [comment]
Yes please!

**Comment by nilscreates** [comment]
Been wanting this pretty much from day 1 and would love to see this. Would speed up custom kit creation right on the deluge immensely, particularly when sampling and slicing. Also w/o needing to go through a PC.

**Comment by darenager** [comment]
When I first got my Deluge some years back the superbly simple implementation of track/clip copy (hold button in row you want to copy, press button in empty row, done) blew my mind, I was a little disappointed to learn that it did not work for copying kit rows though. I always thought it seemed inconsistent, so I’d very much like to see it finally implemented too.

**Comment by sichtbeton** [comment]
this "sort of" works right now in the latest nightly. you can save a kit row as synth via audition+save. then load a kit row via audition + synth. it's 2 steps but basically a copy action. once you done it, it works like charm.

**Reply by thejesiah** [reply]
Hmmmm! Eager to try this out, thanks for the trick!

Oh right, saving a dozen new "synth" patches now... Hmm.. though it's giving the cue to save the synth, it's not giving actually showing it to load in the new kit synth row, just other synths.

**Reply by thejesiah** [reply]
Huh, seems to be a bug. The synths are saving to the projects folder not the synth one.

**Reply by sapphire-arches** [reply]
please file a bug for this, it should be relatively straightforward to fix as long as we have a way to remember to do it =)

**Comment by nilscreates** [comment]
whoop whoop! downloading now!
